src/mixed-line-bar-chart/__tests__/test-chart-series-details.tsx

```diff
@@ -0,0 +1,229 @@
+        toggleState();
+        expectExpanded(false);
+
+        closePopover();
+
+        // Assert that it is still collapsed when opening the popover again
+        openPopover();
+        expectExpanded(false);
+      });
+
+      test('does not render nested items list if the length of sub-items is 0', () => {
+        const { wrapper } = renderChart({
+          ...commonChartProps,
+          detailPopoverSeriesContent: ({ series, y }) => ({
+            key: series.title,
+            value: y,
+            subItems: [],
+          }),
+        });
+
+        wrapper.findApplication()!.focus();
+        const series = wrapper.findDetailPopover()!.findSeries()![0];
+        expect(series.findSubItems()).toHaveLength(0);
+        expect(series.findByClassName(chartSeriesDetailsStyles['.sub-items'])).toBeNull();
+        expect(series.getElement().classList).not.toContain(chartSeriesDetailsStyles['with-sub-items']);
+      });
+    });
+
+    describe('Dev warnings', () => {
+      test('logs a warning when `expandable` is used for a series with no sub-items', () => {
+        const { wrapper } = renderChart({
+          ...commonChartProps,
+          detailPopoverSeriesContent: ({ series, y }) => ({
+            key: series.title,
+            value: y,
+            expandable: true,
+          }),
+        });
+
+        wrapper.findApplication()!.focus();
+        expect(warnOnce).toHaveBeenCalledWith(
+          'MixedLineBarChart',
+          '`expandable` was set to `true` for a series without sub-items. This property will be ignored.'
+        );
+      });
+
+      test('logs a warning and ignores the custom property when a ReactNode is used for an expandable key', () => {
+        const { wrapper } = renderChart({
+          ...commonChartProps,
+          detailPopoverSeriesContent: ({ series, y }) => ({
+            key: <Link>{series.title}</Link>,
+            value: y,
+            expandable: true,
+            subItems: [
+              {
+                key: 'a',
+                value: 1,
+              },
+            ],
+          }),
+        });
+
+        wrapper.findApplication()!.focus();
+        expect(warnOnce).toHaveBeenCalledWith(
+          'MixedLineBarChart',
+          'A ReactNode was used for the key of an expandable series. The series title will be used instead because nested interactive elements can cause accessiblity issues'
+        );
+      });
+
+      test('logs a warning when a ReactNode is used for both key and value', () => {
+        const { wrapper } = renderChart({
+          ...commonChartProps,
+          detailPopoverSeriesContent: ({ series, y }) => ({
+            key: <Link>{series.title}</Link>,
+            value: <Link>{y}</Link>,
+          }),
+        });
+
+        wrapper.findApplication()!.focus();
+        expect(warnOnce).toHaveBeenCalledWith(
+          'MixedLineBarChart',
+          'Use a ReactNode for the key or the value of a series, but not for both'
```

Comment:
This might require additional explanation. The intention is to disallow links on both sides, but the ReactNode does not necessarily mean a link.

Reply:
Done in [3c330ad](https://github.com/cloudscape-design/components/pull/1644/commits/3c330adc6f9a0204174c570c4693dc758977d55a).